lib/LTP/utils.pm

```diff
@@ -402,6 +403,10 @@ sub get_default_pkg {
 }
 
 sub install_from_repo {
+    # Workaround for kernel-64kb, until we add multibuild support to LTP package
+    # Lock kernel-default to don't pull it as LTP dependency
+    zypper_call 'al kernel-default' if get_kernel_flavor eq 'kernel-64kb';
```

Comment:
Would it make sense to lock `kernel-default` for all non-default flavors?

Reply:
Rather not at the moment, as current approach worked fine for other kernels. I will include multibuild support to LTP to cleanup work which I already planned https://progress.opensuse.org/issues/132182.

Reply:
OK